styles/01-settings/brand.css

```diff
@@ -2,30 +2,39 @@
 :root{
   --color-primary: #2e2e2e;
   --color-secondary: #646668;
-  --color-accent: #dd0f4d;
+  --color-accent: #c9003c;
   --color-text: #424242;
   --color-background: #c0b7b7;
 }
 .bg-primary {
   background-color: var(--color-primary);
+  color: #c0b7b7;
 }
 .outline-primary{
-  border: 1px solid var(--color-primary);
+  border: 2px solid var(--color-primary);
 }
 
 .bg-secondary {
   background-color: var(--color-secondary);
+  color: #ffffff;
 }
 .outline-secondary {
-  border: 1px solid var(--color-secondary);
+  border: 2px solid var(--color-secondary);
 } 
 
 .bg-accent {
   background-color: var(--color-accent);
+  color: #ffffff;
```

Comment:
Use variables for color property also.

Reply:
Will note that down for code refactor. Thanks!

Reply:
Done!